context/app/static/js/components/entity-search/facets/select/FacetItemLabelCount/FacetItemLabelCount.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { Flex, FormLabelText } from './style';
+
+function FacetItemLabelCount({ label, count, active, ...rest }) {
+  return (
+    <Flex {...rest} $active={active}>
+      <FormLabelText>{label}</FormLabelText>
```

Comment:
could use `children`... but it doesn't matter.

Reply:
There wasn't a clear child in this case, so I opted to use the 'slots' pattern.